src/server/routes/launchpad.js

```diff
@@ -0,0 +1,12 @@
+import { Router } from 'express';
+import { json } from 'body-parser';
+
+import { newSnap, findSnap } from '../handlers/launchpad';
+
+const router = Router();
+
+router.use('/launchpad/snaps', json());
+router.post('/launchpad/snaps', newSnap);
+router.get('/launchpad/snaps', findSnap);
```

Comment:
@cjwatson Do we have any tests/examples/docs about how to use these routes?

Digging through the code it's possible but quite hard to find what kind of parameters and in what format they expect, and what they return/